packages/dev/core/src/Physics/v2/physicsBody.ts

```diff
@@ -0,0 +1,170 @@
+import type { IPhysicsEnginePluginV2, MassProperties } from "./IPhysicsEnginePluginV2";
+import type { PhysicsShape } from "./physicsShape";
+import { Vector3 } from "../../Maths/math.vector";
+import type { Scene } from "../../scene";
+
+/**
+ *
+ */
+export class PhysicsBody {

```

Comment:
this entire class seems to be a proxy to the physics plugin. Can't it just be a in a form of a bound proxy to the plugin?
Or is it planned to have more logic in this class?

Reply:
I'm not sure yet. I didn't list all the 'small' processing needed when creating this object. For sure Shape will have more logic. idk yet for body and constraint. I guess it will at least contain some checks.